.github/workflows/android-internal-build.yml

```diff
@@ -115,7 +121,8 @@ jobs:
           cd android/scripts
           ./internal-build.sh
         env:
-          MIMOTO_HOST: ${{ github.event.inputs.backendServiceUrl }}
+          MIMOTO_HOST: ${{ github.event.inputs.mimotoBackendServiceUrl }}
+          ESIGNET_HOST: ${{ github.event.inputs.esignetBackendServiceUrl }}
```

Comment:
We should add the esignet env to Fastfile as well so fastlane will have the env context.

ESIGNET_HOST = ENV["ESIGNET_HOST"]